examples/log-collection/fluentd/fluent.conf

```diff
@@ -28,6 +28,7 @@
     read_from_head               true
     follow_inodes                true
     refresh_interval             "#{ENV.fetch('FLUENTD_IN_TAIL_REFRESH_INTERVAL')}"
+    from_encoding                UTF-8
```

Comment:
This is an example file, so my suggestion is to add a comment on an assumption like:

>  input is supposed to be UTF-8 encoded and expect in-tail to emit the input as UTF-8 string

Reply:
Thanks, yokawasa-san. Do you mean like this?

1661c23

Reply:
perfect!